inbm/dispatcher-agent/tests/unit/source/test_ubuntu_source_cmd.py

```diff
@@ -196,7 +196,7 @@ class TestUbuntuApplicationSourceManager:
     def test_add_app_with_gpg_key_successfully(self):
         try:
             params = ApplicationAddSourceParameters(
-                file_name="intel-gpu-jammy.list",
+                source_list_file_name="intel-gpu-jammy.list",
```

Comment:
Update the test cases to reflect the new parameter names and ensure they are still valid after the refactor. [important]